muikku-core-plugins/pom.xml

```diff
@@ -258,6 +258,29 @@
 
   <build>
     <plugins>
+      <plugin>
+        <artifactId>maven-clean-plugin</artifactId>
+        <version>3.3.1</version>
+        <configuration>
+          <filesets>
+            <fileset>
+              <directory>src/main/resources/META-INF/resources/scripts/dist</directory>
+              <includes>
+                <include>**/*</include>
+              </includes>
+              <followSymlinks>false</followSymlinks>
+            </fileset>
+            <fileset>
+              <directory>src/main/resources/META-INF/resources/scripts/src/generated</directory>
+              <includes>
+                <include>**/*</include>
+              </includes>
+              <followSymlinks>false</followSymlinks>
+            </fileset>
+          </filesets>
+        </configuration>
+      </plugin>
+      
```

Comment:
At the moment deploy is done with clean. Not sure how this affects that. Why are we also cleanind dist?

Reply:
Maybe we should remove clean from deploy phase? I don't really see the need for it.